pkg/runtimeproxy/server/cri/criserver.go

```diff
@@ -105,7 +105,7 @@ func (c *RuntimeManagerCriServer) interceptRuntimeRequest(serviceType RuntimeSer
 
 	// pre call hook server
 	// TODO deal with the Dispatch response
-	if response, err := c.hookDispatcher.Dispatch(ctx, runtimeHookPath, config.PreHook, resourceExecutor.GenerateHookRequest()); err != nil {
+	if response, err := c.hookDispatcher.Dispatch(ctx, runtimeHookPath, config.PreHook, resourceExecutor.GenerateHookRequest()); err != nil || response == nil {
```

Comment:
no need to log error here in this case? WDYT？

Reply:
@ZYecho I have no idea, @honpey WDYT？

Reply:
I'm fine with either way.

```go
response, err := c.hookDispatcher.Dispatch(ctx, runtimeHookPath, config.PreHook, resourceExecutor.GenerateHookRequest())
if err != nil {
	// say something
}
if response == nil {
	// say something
}
err = resourceExecutor.UpdateRequest(response, request)
if err != nil {
	// say something
}
```

Reply:
yeah, add some comments about when the response is nil?

Reply:
@hormes ok

Reply:
Done @hormes @ZYecho @jasonliu747